x-pack/platform/packages/shared/alerting-rule-utils/src/get_ecs_groups.test.ts

```diff
@@ -63,3 +63,32 @@ describe('getEcsGroups', () => {
     expect(getEcsGroups(groups)).toEqual({ tags: ['abc'] });
   });
 });
+
+describe('getEcsGroupsFromGrouping', () => {
+  it('should return all groups if they are string ecs fields', () => {
+    const grouping = { ['host.name']: 'host-1', ['container.id']: 'container-1' };
```

Comment:
@cesco-f I created a similar function and [here's](https://github.com/elastic/kibana/pull/228754/files#diff-bb71b6f7f18d3cd824ee950e77c43dc6d0fefd38f6388472428935fe4dadf60dR12) the format of the `groupings` object I used

```
const groupings = {
      host: {
        name: 'host-1',
      },
      container: {
        id: 'container-1',
      },
    };
```

@benakansara Can you clarify what is the format you had in mind?

Reply:
I was passing the flatten version of the object because it's easier to check the ecs field this way and we avoid flattening the object again.

But I'm ok also with your implementation.

Reply:
I talked with @benakansara and I will change my implementation to pass the flattened version.

Reply:
@mgiota @cesco-f In the custom threshold rule, I used the object structure of `kibana.alert.grouping` to match what we save in context variables. Does it mean we have a mismatch now in how we saved this data?